ckanext/restricted/controller.py

```diff
@@ -399,7 +399,7 @@ def restricted_request_organization_form(self, data=None, errors=None,
         data['user_email'] = user.get('email', '')
 
         extra_vars = {
-            'data': data, 'group_type': "organization",
+            'data': data, 'group_type': "organization", 'pkg_dict': {},
```

Comment:
Sorry, my bad. My comment was about this part. `restricted_request_organization_form.html'` is extending `page.html` not `package/page.html` so we don't need it here.